data/remnant outfits.txt

```diff
@@ -454,5 +454,5 @@ outfit "Remnant License"
 
 outfit "Remnant Capital License"
 	category "Special"
-	thumbnail "outfit/unknown"
+	thumbnail "outfit/Remnant Capital License"
```

Comment:
Capital letters in an image name? What is this heresy?

Reply:
I fixed the heresy.